pkgs/development/compilers/intel-graphics-compiler/default.nix

```diff
@@ -64,7 +81,9 @@ stdenv.mkDerivation rec {
   '';
 
   cmakeFlags = [
+    "-Wno-dev"
     "-DVC_INTRINSICS_SRC=${vc_intrinsics_src}"
+    "-DIGC_OPTION__SPIRV_TOOLS_MODE=Prebuilds"
     "-DINSTALL_SPIRVDLL=0"
```

Comment:
CMake Warning:
  Manually-specified variables were not used by the project:
    ...
    INSTALL_SPIRVDLL

Reply:
Done in a follow-up